src/GameHUD.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./App.css";
+import { useParams } from "react-router-dom";
+
+interface IGameHUD {
+  users: string[];
+}
+
+const GameHUD = (props: IGameHUD) => {
+  const { id } = useParams();
+  return (
+    <>
```

Comment:
Feels like the HUD is semantically a thing, so I'd argue it should be wrapped in a div rather than a fragment